lnd.go

```diff
@@ -191,8 +194,8 @@ func lndMain() error {
 	// We wait until the user provides a password over RPC. In case lnd is
 	// started with the --noencryptwallet flag, we use the default password
 	// "hello" for wallet encryption.
```

Comment:
Comment can be changed to just somehting like ".... we use default passwords for wallet encyption"